lib/main.dart

```diff
@@ -1,115 +1,117 @@
-import 'package:flutter/material.dart';
+import "package:application_form/response_page.dart";
+import "package:flutter/material.dart";
+import "package:application_form/assets/constants.dart";
+
+import "home_page.dart";
 
 void main() {
-  runApp(const MyApp());
+  runApp(ApplicationForm());
 }
 
-class MyApp extends StatelessWidget {
-  const MyApp({super.key});
-
-  // This widget is the root of your application.
+class ApplicationForm extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return MaterialApp(
-      title: 'Flutter Demo',
       theme: ThemeData(
```

Comment:
Please create a separate  class for this theme.

Reply:
Changes Done.